.storybook/preview.tsx

```diff
@@ -15,38 +15,38 @@ import { StorybookHeading } from "./storybookComponents";
 // Custom viewport options
 const customViewports = {
   sm: {
-    name: "sm (320px)",
+    name: "sm - small mobile (less than 480)",
     styles: {
-      width: "320px",
-      height: "568px",
+      width: "390px",
```

Comment:
It's intentional to not have 480 or 479 but any width less than 480?

Reply:
Yes.  I set the Storybook preview sizes to align with the new Figma art board sizes.  And those sizes were setup as good working sizes within the breakpoint ranges.